src/app/donation-start/donation-start.component.scss

```diff
@@ -100,6 +100,18 @@
   font-size: 22px;
 }
 
+.c-your-donation__form-field-to-tbg {
```

Comment:
@tbg-jasonfung the QA feedback was around making these consistent with each other, so couldn't they just share a class?

Reply:
@NoelLH think the change is to make the tip input field smaller as outlined by Babsy

Reply:
@NoelLH sorry I just realised I tagged the wrong ticket, keep doing that recently...see this ticket: https://thebiggive.atlassian.net/browse/DON-307

Reply:
Hm, OK. This seems to be the opposite of the suggestion on DON-308 but I agree it makes sense for it to be smaller.